package-lock.json

```diff
@@ -8175,14 +8174,12 @@
         "balanced-match": {
           "version": "1.0.0",
           "bundled": true,
-          "dev": true,
-          "optional": true
```

Comment:
@dawidzarzycki a bit confused why the lock file has updated at all if there's no dependency changes?